src/index.html

```diff
@@ -11,6 +11,17 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search_top">
+      <img class="search_img" src="images/Search.svg"
+       alt="search">
+      <input data-qa="big" class="search_big"
+      type="search" placeholder="Try “Los Angeles”">
```

Comment:
Need to fix code style everywhere
```suggestion
      <input
        data-qa="big"
        class="search_big"
        type="search"
        placeholder="Try “Los Angeles”"
      >
```